### ISSUE TITLE
Test json files are broken / has wrong syntax

### ISSUE THREAD

**Issue by StrongestNumber9** [teragrep/pth_10#338]
**Describe the bug**

```
$ for file in src/test/resources/*.json; do jq 'empty' $file; done;
jq: parse error: Expected value before ',' at line 1, column 196
jq: parse error: Expected value before ',' at line 1, column 187
jq: parse error: Expected value before ',' at line 2, column 315
jq: parse error: Expected value before ',' at line 1, column 174
jq: parse error: Expected value before ',' at line 1, column 198
jq: parse error: Expected value before ',' at line 1, column 198
jq: parse error: Expected value before ',' at line 1, column 480
jq: parse error: Expected value before ',' at line 1, column 500
<snip>
```

**Expected behavior**

Syntax is valid

**How to reproduce**

Run the json files through a validator

**Screenshots**

 
**Software version**

7.1.0
 
**Desktop (please complete the following information if relevant):**
 - OS: 
 - Browser: 
 - Version: 
 
**Additional context**

**Comment by eemhu** [comment]
Please note that DataStreamReader uses JSON Lines by default, meaning each line it is its own JSON record. To use standard JSON, multiLine option should be used.

**Comment by 51-code** [comment]
For now the schema is first read from the json and then afterwards the data is read. The multiLine option has to be set to both of these for it to have effect.

**Comment by StrongestNumber9** [comment]
Seems to be closed in #351
